saclebot_main_pose_config:
    X : 0.0
    Y : 0.0
    Z : 0.0
    roll : 0.0
    pitch : 0.0
    yaw : 0.0

saclebot_red_paper_cup_pose_config:
    X : 1.5
    Y : 1.5
    Z : 0.108
    roll : 0.0
    pitch : 0.0
    yaw : 0.0

saclebot_green_paper_cup_pose_config:
    X : 0.0
    Y : 0.0
    Z : 0.108
    roll : 0.0
    pitch : 0.0
    yaw : 0.0

saclebot_coke_can_pose_config:
    X : 0.0
    Y : 0.0
    Z : -0.08
    roll : 0.0
    pitch : 0.0
    yaw : 0.0

saclebot_dust_pose_config:
    X : 0.0
    Y : 0.0
    Z : 0.0
    roll : 0.0
    pitch : 0.0
    yaw : 0.0
